packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -5,9 +5,18 @@
 import React from 'react';
 import AppLayout from '../AppLayout';
 import AppErrorBoundary from '@fxa-components/AppErrorBoundary';
+import { QueryParams } from '../../lib/types';
+import FlowEvents from '../../lib/flow-event';
 import './index.scss';
 
-export const App = () => {
+type AppProps = {
+  queryParams: QueryParams;
+};
+
+export const App = ({ queryParams }: AppProps) => {
+  const flow = FlowEvents.init(queryParams, true);
+  console.log('Flow data', flow);
```

Comment:
It's here because this is still a draft?

Reply:
Gone now. I had originally not set up anything beyond parsing the incoming params, but I decided to add the full amplitude lib setup.